src/_pytest/cacheprovider.py

```diff
@@ -140,6 +147,10 @@ def _ensure_supporting_files(self):
                 msg = u"# Created by pytest automatically.\n*"
                 gitignore_path.write_text(msg, encoding="UTF-8")
 
+            cachedir_tag_path = self._cachedir.joinpath("CACHEDIR.TAG")
+            if not cachedir_tag_path.is_file():
+                cachedir_tag_path.write_text(CACHEDIR_TAG_CONTENT, encoding="UTF-8")
```

Comment:
i believe on windows this alters the content of the signature file, i propose using a bytes literal and write_bytes

Reply:
Fixed